src/helpers.js

```diff
@@ -4,6 +4,8 @@ import { scalePow } from 'd3-scale'
 import chroma from 'chroma-js'
```

Comment:
helpers is a bunch of useful fucntions around app just central place to have them